addons/l10n_id/tests/test_qris.py

```diff
@@ -111,26 +110,27 @@ def test_qris_qr_code_generation(self):
             patched.assert_not_called()
 
             # Check that if the qr code has expired, we correctly generate a new one.
-            qr_details[0]['qris_creation_datetime'] = '2024-02-27 03:00:00'
-            self.qris_qr_invoice.l10n_id_qris_invoice_details = qr_details
+            qr_details.qris_creation_datetime = '2024-02-27 03:00:00'
             self.qris_qr_invoice.with_context({'is_online_qr': True})._generate_qr_code()
             patched.assert_called_once()
 
+            self.assertEqual(len(self.qris_qr_invoice.l10n_id_qris_transactions), 2)
+
         with patch(
             'odoo.addons.l10n_id.models.res_bank._l10n_id_make_qris_request', return_value=self.success_qris_get_second
         ):
-            qr_details = self.qris_qr_invoice.l10n_id_qris_invoice_details
+            qr_details = self.qris_qr_invoice.l10n_id_qris_transactions
             qr_details[-1]['qris_creation_datetime'] = '2024-02-27 03:00:00'
-            self.qris_qr_invoice.l10n_id_qris_invoice_details = qr_details
+            # self.qris_qr_invoice.l10n_id_qris_transaction.qris_invoice_details = qr_details
```

Comment:
You can remove, it was needed before due to the json field but it is no longer the case